test/runtests.jl

```diff
@@ -840,3 +850,11 @@ end
         @test x == [3]
     end
 end
+
+####
+#### static analysis with JET
+####
+
+import JET
+@test isempty(JET.get_reports(JET.report_package(TransformVariables;
+                                                 target_modules = (TransformVariables,))))
```

Comment:
A bit simpler (but equivalent IIRC) would be
```suggestion
JET.test_package(TransformVariables; target_modules = (TransformVariables,))
```